server/src/test/integration/org/apache/cassandra/sidecar/testing/CassandraSidecarTestContext.java

```diff
@@ -73,7 +74,8 @@ public class CassandraSidecarTestContext implements AutoCloseable
     private final AbstractCassandraTestContext abstractCassandraTestContext;
     private final Vertx vertx;
     private final List<InstancesMetadataListener> instancesMetadataListeners;
-    private int numInstancesToManage;
+    // array of nodeNums that are 1-based
+    private int[] instancesToManage = null;
```

Comment:
it makes the Cassandra instances flexible. Now, one can choose to manage, instance 1 and 3.